pkg/kubelet/kubelet_pods_test.go

```diff
@@ -35,6 +35,7 @@ import (
 	core "k8s.io/client-go/testing"
 	"k8s.io/client-go/tools/record"
 	"k8s.io/kubernetes/pkg/api"
+	_ "k8s.io/kubernetes/pkg/api/install"
```

Comment:
Can you add a TODO here saying we can remove this import if we change all `api.Registry.GroupOrDie(v1.GroupName).GroupVersion.String(),` to `"v1"`?